src/pages/MainPage/PrivateRoomModal.tsx

```diff
@@ -0,0 +1,60 @@
+import * as Dialog from '@radix-ui/react-dialog';
+import { Cross2Icon, PaperPlaneIcon } from '@radix-ui/react-icons';
+import { ComponentProps, ReactNode, useState } from 'react';
+import Input from '@/common/Input/Input';
+
+interface PrivateRoomModalProps {
+  children: ReactNode;
+  className?: ComponentProps<'div'>['className'];
+}
+
+const wait = () => new Promise((resolve) => setTimeout(resolve, 2000));
+
+const PrivateRoomModal = ({ children, className }: PrivateRoomModalProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  return (
+    <Dialog.Root
+      open={isOpen}
+      onOpenChange={setIsOpen}>
+      <Dialog.Trigger>{children}</Dialog.Trigger>
+      <Dialog.Portal>
+        <Dialog.Overlay
+          className={
+            'fixed top-1/2 left-1/2 translate-x-[-50%] translate-y-[-50%] w-[100vw] h-[100vh] bg-black/30 animate-[overlayShow_150ms_cubic-bezier(0.16,1,0.3,1)]'
```

Comment:
```suggestion
            'fixed inset-0 w-[100vw] h-[100vh] bg-black/30 animate-[overlayShow_150ms_cubic-bezier(0.16,1,0.3,1)]'
```
이렇게 줄일 수 있던데 어떠신가용?! 😊

Reply:
와 대박이네요 감사합니다ㅎㅎ
`translate-x-[-50%] translate-y-[-50%]`속성도 inset에 포함되는건가요?

Reply:
말씀해주신대로 inset은 단순히 top left bottom right의 축약이네요!! 😊